data/munin/groonga_n_records_

```diff
@@ -169,7 +172,39 @@ def setup
   setup_gqtp
 end
 
+def pid_files_esists?
```

Comment:
We should use `exist?` not `exists?` like `File.exist?`. (`File.exists?` is deprecated.)

Reply:
Oh.. I see.